postgre-client/src/tasks.rs

```diff
@@ -93,51 +97,80 @@ impl PgClient {
         }
     }
 
-    pub async fn update_tasks_and_attempts(
-        &self,
-        data: Vec<UpdatedTask>,
-    ) -> Result<(), IndexDbError> {
-        let mut query_builder: QueryBuilder<'_, Postgres> =
-            QueryBuilder::new("UPDATE tasks SET tsk_status = tmp.tsk_status, tsk_attempts = tsk_attempts+1, tsk_error = tmp.tsk_error FROM (");
+    pub async fn update_tasks(&self, data: Vec<UpdatedTask>) -> Result<(), IndexDbError> {
+        let mut query_builder: QueryBuilder<'_, Postgres> = QueryBuilder::new(
+            "
+            UPDATE tasks 
+            SET 
+                task_status = tmp.task_status, 
+                etag = tmp.etag, 
+                last_modified_at = tmp.last_modified_at, 
+                mutability = tmp.mutability, 
+                next_refresh_at = NOW + INTERVAL '1 day' 
```

Comment:
Are we setting the next refresh/attempt time for immutable as well?

Reply:
We aren't